python/example/network_ring.py

```diff
@@ -109,9 +109,8 @@ def global_properties(self, kind):
 recipe = ring_recipe(ncells)
 
 # (12) Create an execution context using all locally available threads, domain decomposition and simulation
-context = arbor.context("avail_threads")
-decomp = arbor.partition_load_balance(recipe, context)
-sim = arbor.simulation(recipe, decomp, context)
+ctx = arbor.context("avail_threads")
+sim = arbor.simulation(recipe, context=ctx)
```

Comment:
```suggestion
sim = arbor.simulation(recipe, arbor.context("avail_threads"))
```

Reply:
Robin requested the exact opposite ;)

Reply:
The main thing I care about is that as of The Swap(tm) `context=` can be left out. I'd have peace with
```suggestion
sim = arbor.simulation(recipe, ctx)
```

Reply:
So, I'll let you two battle that out, I'd chosen @brenthuisman 's suggestion and only change due to Robin's wishes.
Less typing is almost always to be preferred.

Reply:
What I meant is: no need to pass with the `context` key anymore. Second arg is expected to be a `arbor.context`.

Reply:
Fine with me :)